src/include/executor/tuptable.h

```diff
@@ -140,6 +148,8 @@ typedef struct TupleTableSlot
 
     /* System attributes */
     Oid         tts_tableOid;
+
+    SlotDeformTupleCodeGenInfo slot_deform_tuple_gen_info;
```

Comment:
I have two issues here:
* Even if we do not codegen, we allocate memory sizeof(SlotDeformTupleCodeGenInfo). 
* In the future, we will have numerous codegened functions, whose function pointer can be reached through slot. Are we going to allocate memory for all structs of the new functions here as well?